Formula/liquid-dsp.rb

```diff
@@ -0,0 +1,45 @@
+class LiquidDsp < Formula
+  desc "Signal processing (DSP) library for software-defined radios"
+  homepage "http://liquidsdr.org/"
+  url "https://github.com/jgaeddert/liquid-dsp/archive/v1.2.0.tar.gz"
+  sha256 "783854b63f5e9a9830dcd95b2c1d5c0bef7d10d22a0e27844a354b59283b6de3"
+  head "https://github.com/jgaeddert/liquid-dsp.git"
+
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "fftw" => :recommended
+
+  # Fix configure checks to be able to compile with LLVM/Clang
```

Comment:
Some conflicts that need resolved upstream on this. Would like to see upstream accept this and at least implicitly state they are wanting to support Clang before merging this.